jdi-light-mobile-examples/src/main/java/selenium/seleniumPO/HomePage.java

```diff
@@ -0,0 +1,43 @@
+package selenium.seleniumPO;
+
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import org.openqa.selenium.By;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+import static com.epam.jdi.light.driver.WebDriverFactory.getDriver;
+
+public class HomePage {
+    private String url = "https://jdi-testing.github.io/jdi-light/index.html";
+    public void open() {
+        getDriver().navigate().to(url);
+    }
+
+    @FindBy(css = ".sidebar-menu [ui=label]") List<WebElement> navigation;
+    public void navigateTo(int num) {
+        try {
+            navigation.get(num).click();
+        } catch (StaleElementReferenceException ex) {
+            getDriver().findElements(By.cssSelector(".sidebar-menu [ui=label]")).get(num).click();
+        }
+    }
+    public void navigateTo(String text) {
+        List<WebElement> navigations = getDriver().findElements(By.cssSelector(".sidebar-menu [ui=label]"));
+        for (WebElement el : navigations) {
+            if (el.getText().equals(text)) {
+                el.click();
+                return;
+            }
+        }
+    }
+    @FindBy(css = ".profile-photo") WebElement profilePhoto;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)